go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/stretchr/objx v0.1.2-0.20180626195558-9e1dfc121bca // indirect
 	github.com/stretchr/testify v1.6.1
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975 // indirect
+	gopkg.in/yaml.v2 v2.2.8
```

Comment:
have you run as well `go mod vendor`? Because I do not see the new dependency vendored

Reply:
yes , it is already there and in go.sum . it's wired that it was not listed as indirect here.